### DISCUSSION TITLE
Fight sticks ps3/ps4

### DISCUSSION THREAD

**Discussion by sotojo01** [cathery/sys-con#129] [Controller requests]
Razer panthera Ps3/ps4 fight stick: (not the "evo" variant)

**Description of the controller**
it supports usb only and has a ps3 and ps4 toggle switch (thus its mapped that way even has a touchpad) , no bluetooth support. has only one analog stick since its an arcade style fight stick. no rumble.  analog stick can be programmed to act as the left or right stick or dpad via a toggle switch on controller.

**Screenshot of the device**

![2020011811453300-DB1426D1DFD034027CECDE9C2DD914B8](https://user-images.githubusercontent.com/49783524/72667367-24748280-39e9-11ea-8d75-8aff088c6d32.jpg)

**Additional context**
Tried to change global settings from true to false and running fight stick on ps3 and ps4 mode and it didn’t work.  Would be great to have fight stick support since they are mostly usb and not many switch compatible ones. Would love this to be updated. I used a razer panthera fight stick in my test.

link for additional info:
https://www.razer.com/gaming-controllers/razer-dragon-ball-fighterz-panthera

 https://www.amazon.com/Razer-Panthera-Mod-Capable-Compartment-playstation-4/dp/B01N1KFRU7

**Comment by sotojo01** [comment]
Are any fight stick supported?

**Comment by cathery** [comment]
sys-con only supports DS3, DS4 and XInput controllers at the moment sorry
